src/Features/TreeBrowse.php

```diff
@@ -61,7 +61,7 @@ public static function showBrowseView(string $itemtype, array $params, $update =
         global $CFG_GLPI;
 
         $ajax_url    = $CFG_GLPI["root_doc"] . "/ajax/treebrowse.php";
-        $loading_txt = addslashes(__('Loading...'));
+        $loading_txt = addslashes(__s('Loading...'));
```

Comment:
`addslashes` is not needed anymore.
```suggestion
        $loading_txt = __s('Loading...');
```